drizzlepac/pars/hap_pars/default_parameters/wfc3/ir/wfc3_ir_quality_control_all.json

```diff
@@ -1,3 +1,28 @@
 {
-    "placeholder": "QC"
+    "ci filter": {
+      "aperture": {
+        "bthresh": 5.0,
+        "ci_lower_limit": 0.25,
+        "ci_upper_limit": 0.55
+      },
+      "segment": {
+        "bthresh": 5.0,
+        "ci_lower_limit": 0.25,
+        "ci_upper_limit": 0.55
+      }
+    },
+    "swarm filter": {
+      "upper_epp_limit": 70000.0,
+      "lower_epp_limit": 2000.0,
+      "eppsky_limit": 1000.0,
+      "selfradius": 10.0,
+      "cuts_list": [2000000.0, 1800000.0, 1000000.0, 500000.0, 70000.0, 20000.0, 0.0],
+      "selfradii_list": [25.0, 100.0, 35.0, 30.0, 20.0, 15.0, 10.0],
```

Comment:
Observation: I only see cuts_list and selfradii_list in the WFC3 IR default_parameters and user_parameters files.  Are these special parameters for WFC3 IR or are these deprecated variables?

Reply:
hey @mdlpstsci , "cuts_list" and "selfradii_list" are special WFC3/IR-only parameters.